### ISSUE TITLE
Dalamud not loading on startup

### ISSUE THREAD

**Issue by Kaomojis** [goatcorp/Dalamud#1957]
Have a issue where Dalamud no longer starts up when the game loads, no matter how many times I restart. The plugins and settings option are still in the menu, but selecting them does nothing. I've deleted the %AppData% folder for XIV launcher and re-installed Dalamud, but the issue still persists. This is what I see when I pull up dalamud.log:

> 2024-07-21 23:52:16.060 -07:00 [INF] --------------------------------------------------------------------------------
> 2024-07-21 23:52:16.060 -07:00 [INF] Initializing a session..
> 2024-07-21 23:52:19.161 -07:00 [INF] Trying to set up Loc for culture en
> 2024-07-21 23:52:19.162 -07:00 [INF] This is Dalamud - Core: 1.0.0.5-0-gc88689a7, CS: 07f7b3fd [4724]
> 2024-07-21 23:52:19.188 -07:00 [INF] [PLUGINM] Now reloading all PluginMasters...
> 2024-07-21 23:52:19.190 -07:00 [INF] [PLUGINR] Fetching repo: https://kamori.goats.dev/Plugin/PluginMaster
> 2024-07-21 23:52:19.257 -07:00 [INF] [PLUGINM] [T3] PM repos OK!
> 2024-07-21 23:52:19.267 -07:00 [INF] LASTEXCEPTION:eyJXaGVuIjoiMjAyNC0wNy0yMVQyMzo1MjoxOS4yNTAxODU4LTA3OjAwIiwiSW5mbyI6IlN5c3RlbS5BZ2dyZWdhdGVFeGNlcHRpb246IEEgVGFzaydzIGV4Y2VwdGlvbihzKSB3ZXJlIG5vdCBvYnNlcnZlZCBlaXRoZXIgYnkgV2FpdGluZyBvbiB0aGUgVGFzayBvciBhY2Nlc3NpbmcgaXRzIEV4Y2VwdGlvbiBwcm9wZXJ0eS4gQXMgYSByZXN1bHQsIHRoZSB1bm9ic2VydmVkIGV4Y2VwdGlvbiB3YXMgcmV0aHJvd24gYnkgdGhlIGZpbmFsaXplciB0aHJlYWQuIChPbmUgb3IgbW9yZSBlcnJvcnMgb2NjdXJyZWQuIChPbmUgb3IgbW9yZSBlcnJvcnMgb2NjdXJyZWQuIChGYWlsZWQgdG8gZmluZCB0aGUgYXNzZXQgTG9kZXN0b25lR2FtZVN5bWJvbC4pKSlcclxuIC0tLT4gU3lzdGVtLkFnZ3JlZ2F0ZUV4Y2VwdGlvbjogT25lIG9yIG1vcmUgZXJyb3JzIG9jY3VycmVkLiAoT25lIG9yIG1vcmUgZXJyb3JzIG9jY3VycmVkLiAoRmFpbGVkIHRvIGZpbmQgdGhlIGFzc2V0IExvZGVzdG9uZUdhbWVTeW1ib2wuKSlcclxuIC0tLT4gU3lzdGVtLkFnZ3JlZ2F0ZUV4Y2VwdGlvbjogT25lIG9yIG1vcmUgZXJyb3JzIG9jY3VycmVkLiAoRmFpbGVkIHRvIGZpbmQgdGhlIGFzc2V0IExvZGVzdG9uZUdhbWVTeW1ib2wuKVxyXG4gLS0tPiBTeXN0ZW0uSU8uRmlsZU5vdEZvdW5kRXhjZXB0aW9uOiBGYWlsZWQgdG8gZmluZCB0aGUgYXNzZXQgTG9kZXN0b25lR2FtZVN5bWJvbC5cclxuRmlsZSBuYW1lOiAnTG9kZXN0b25lR2FtZVN5bWJvbCdcclxuICAgYXQgRGFsYW11ZC5TdG9yYWdlLkFzc2V0cy5EYWxhbXVkQXNzZXRNYW5hZ2VyLjw+Y19fRGlzcGxheUNsYXNzMTlfMC48PENyZWF0ZVN0cmVhbUFzeW5jPmdfX0NyZWF0ZUlubmVyQXN5bmN8MT5kLk1vdmVOZXh0KCkgaW4gQzpcXGdvYXRzb2Z0XFxjb21wYW55c2VjcmV0c1xcZGFsYW11ZFxcXFxTdG9yYWdlXFxBc3NldHNcXERhbGFtdWRBc3NldE1hbmFnZXIuY3M6bGluZSAyNDlcclxuICAgLS0tIEVuZCBvZiBpbm5lciBleGNlcHRpb24gc3RhY2sgdHJhY2UgLS0tXHJcbiAgIGF0IERhbGFtdWQuU3RvcmFnZS5Bc3NldHMuRGFsYW11ZEFzc2V0TWFuYWdlci48PmNfX0Rpc3BsYXlDbGFzczE5XzAuPDxDcmVhdGVTdHJlYW1Bc3luYz5nX19DcmVhdGVJbm5lckFzeW5jfDE+ZC5Nb3ZlTmV4dCgpIGluIEM6XFxnb2F0c29mdFxcY29tcGFueXNlY3JldHNcXGRhbGFtdWRcXFxcU3RvcmFnZVxcQXNzZXRzXFxEYWxhbXVkQXNzZXRNYW5hZ2VyLmNzOmxpbmUgMjc3XHJcbiAgIC0tLSBFbmQgb2YgaW5uZXIgZXhjZXB0aW9uIHN0YWNrIHRyYWNlIC0tLVxyXG4gICAtLS0gRW5kIG9mIGlubmVyIGV4Y2VwdGlvbiBzdGFjayB0cmFjZSAtLS0iLCJDb250ZXh0IjoiVW5vYnNlcnZlZCBleGNlcHRpb24gaW4gVGFzay4ifQ==
> 2024-07-21 23:52:19.228 -07:00 [ERR] Unobserved exception in Task.
> System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (One or more errors occurred. (One or more errors occurred. (Failed to find the asset LodestoneGameSymbol.)))
>  ---> System.AggregateException: One or more errors occurred. (One or more errors occurred. (Failed to find the asset LodestoneGameSymbol.))
>  ---> System.AggregateException: One or more errors occurred. (Failed to find the asset LodestoneGameSymbol.)
>  ---> System.IO.FileNotFoundException: Failed to find the asset LodestoneGameSymbol.
> File name: 'LodestoneGameSymbol'
>    at Dalamud.Storage.Assets.DalamudAssetManager.<>c__DisplayClass19_0.<<CreateStreamAsync>g__CreateInnerAsync|1>d.MoveNext() in C:\goatsoft\companysecrets\dalamud\\Storage\Assets\DalamudAssetManager.cs:line 249
>    --- End of inner exception stack trace ---
>    at Dalamud.Storage.Assets.DalamudAssetManager.<>c__DisplayClass19_0.<<CreateStreamAsync>g__CreateInnerAsync|1>d.MoveNext() in C:\goatsoft\companysecrets\dalamud\\Storage\Assets\DalamudAssetManager.cs:line 277
>    --- End of inner exception stack trace ---
>    --- End of inner exception stack trace ---
> 2024-07-21 23:52:19.313 -07:00 [INF] [PLUGINM] [T3] PMC OK!
> 2024-07-21 23:52:19.313 -07:00 [INF] [PLUGINM] [T3] PML OK!
> 2024-07-21 23:52:19.325 -07:00 [INF] TROUBLESHOOTING:eyJMb2FkZWRQbHVnaW5zIjpbXSwiUGx1Z2luU3RhdGVzIjp7fSwiRXZlclN0YXJ0ZWRMb2FkaW5nUGx1Z2lucyI6W10sIkRhbGFtdWRWZXJzaW9uIjoiMTAuMC4wLjUiLCJEYWxhbXVkR2l0SGFzaCI6IjEuMC4wLjUtMC1nYzg4Njg5YTciLCJHYW1lVmVyc2lvbiI6IjIwMjQuMDcuMTAuMDAwMS4wMDAwIiwiTGFuZ3VhZ2UiOiJFbmdsaXNoIiwiRG9EYWxhbXVkVGVzdCI6ZmFsc2UsIkJldGFLZXkiOm51bGwsIkRvUGx1Z2luVGVzdCI6ZmFsc2UsIkxvYWRBbGxBcGlMZXZlbHMiOmZhbHNlLCJJbnRlcmZhY2VMb2FkZWQiOmZhbHNlLCJGb3JjZWRNaW5Ib29rIjpmYWxzZSwiVGhpcmRSZXBvIjpbXSwiSGFzVGhpcmRSZXBvIjpmYWxzZX0=
> 2024-07-21 23:52:19.351 -07:00 [INF] [PLUGINM] ============= LoadPluginsSync(AnytimeSync) START =============
> 2024-07-21 23:52:19.351 -07:00 [INF] [PLUGINM] ============= LoadPluginsSync(AnytimeSync) END =============
> 2024-07-21 23:52:19.351 -07:00 [INF] [PLUGINM] ============= LoadPluginsAsync(AnytimeAsync) START =============
> 2024-07-21 23:52:19.353 -07:00 [INF] [PLUGINM] ============= LoadPluginsAsync(AnytimeAsync) END =============
> 2024-07-21 23:52:19.574 -07:00 [INF] Lumina is ready: C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack
> 2024-07-21 23:52:20.143 -07:00 [INF] [PLUGINR] Successfully fetched repo: https://kamori.goats.dev/Plugin/PluginMaster
> 2024-07-21 23:52:20.144 -07:00 [INF] [PLUGINM] PluginMasters reloaded, now refiltering...
> 2024-07-21 23:52:22.102 -07:00 [ERR] Failed to get ReShade 6 DXGISwapChain::on_present offset!
> 2024-07-21 23:52:22.103 -07:00 [INF] Hooking present for ReShade with fallback 5.X sig
> 2024-07-21 23:52:22.117 -07:00 [INF] ReShade DLL: C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\dxgi.dll (crosire's ReShade post-processing injector for 64-bit - 6.2.0.1881) with DXGISwapChain::on_present at 0x7FFDFAC554BA(C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\dxgi.dll+0x1254BA(.text+0x1244BA))
> 2024-07-21 23:52:24.094 -07:00 [INF] [PLUGINM] ============= LoadPluginsSync(FrameworkTickSync) START =============
> 2024-07-21 23:52:24.094 -07:00 [INF] [PLUGINM] ============= LoadPluginsSync(FrameworkTickSync) END =============
> 2024-07-21 23:52:24.094 -07:00 [INF] [PLUGINM] ============= LoadPluginsAsync(FrameworkTickAsync) START =============
> 2024-07-21 23:52:24.095 -07:00 [INF] [PLUGINM] ============= LoadPluginsAsync(FrameworkTickAsync) END =============
> 2024-07-21 23:52:24.202 -07:00 [WRN] [HITCH] Long FrameworkUpdate detected, 157.1211ms > 50ms - check in the plugin stats window.

**Comment by Soreepeong** [comment]
As a workaround for the time being, try disabling ReShade; you can rename `dxgi.dll` to `dxgi.dll.old` in the game installation directory.

**Comment by Alpacamutti** [comment]
Got the same issues! works fine when you disabling GeShade but can't work with each other rn. A friend use ReShade and it works fine, so its seems like more a GShade problem?

**Comment by fdinsen** [comment]
> Got the same issues! works fine when you disabling GeShade but can't work with each other rn. A friend use ReShade and it works fine, so its seems like more a GShade problem?

It isn't just related to GShade. I'm having the same problem, but I'm using ReShade. 

My dalamud.log file is:

```
2024-07-22 22:57:16.032 +02:00 [INF] --------------------------------------------------------------------------------
2024-07-22 22:57:16.032 +02:00 [INF] Initializing a session..
2024-07-22 22:57:16.706 +02:00 [INF] This is Dalamud - Core: 1.0.0.5-0-gc88689a7, CS: 07f7b3fd [4724]
2024-07-22 22:57:16.827 +02:00 [INF] [PLUGINM] Now reloading all PluginMasters...
2024-07-22 22:57:16.829 +02:00 [INF] [PLUGINR] Fetching repo: https://kamori.goats.dev/Plugin/PluginMaster
2024-07-22 22:57:16.891 +02:00 [INF] [PLUGINM] [T3] PM repos OK!
2024-07-22 22:57:17.138 +02:00 [INF] [PLUGINM] [T3] PMC OK!
2024-07-22 22:57:17.138 +02:00 [INF] [PLUGINM] [T3] PML OK!
2024-07-22 22:57:17.166 +02:00 [INF] TROUBLESHOOTING:eyJMb2FkZWRQbHVnaW5zIjpbXSwiUGx1Z2luU3RhdGVzIjp7fSwiRXZlclN0YXJ0ZWRMb2FkaW5nUGx1Z2lucyI6W10sIkRhbGFtdWRWZXJzaW9uIjoiMTAuMC4wLjUiLCJEYWxhbXVkR2l0SGFzaCI6IjEuMC4wLjUtMC1nYzg4Njg5YTciLCJHYW1lVmVyc2lvbiI6IjIwMjQuMDcuMTAuMDAwMS4wMDAwIiwiTGFuZ3VhZ2UiOiJFbmdsaXNoIiwiRG9EYWxhbXVkVGVzdCI6ZmFsc2UsIkJldGFLZXkiOm51bGwsIkRvUGx1Z2luVGVzdCI6ZmFsc2UsIkxvYWRBbGxBcGlMZXZlbHMiOmZhbHNlLCJJbnRlcmZhY2VMb2FkZWQiOmZhbHNlLCJGb3JjZWRNaW5Ib29rIjpmYWxzZSwiVGhpcmRSZXBvIjpbXSwiSGFzVGhpcmRSZXBvIjp0cnVlfQ==
2024-07-22 22:57:17.195 +02:00 [INF] [PLUGINM] ============= LoadPluginsSync(AnytimeSync) START =============
2024-07-22 22:57:17.195 +02:00 [INF] [PLUGINM] ============= LoadPluginsSync(AnytimeSync) END =============
2024-07-22 22:57:17.196 +02:00 [INF] [PLUGINM] ============= LoadPluginsAsync(AnytimeAsync) START =============
2024-07-22 22:57:17.198 +02:00 [INF] [PLUGINM] ============= LoadPluginsAsync(AnytimeAsync) END =============
2024-07-22 22:57:17.298 +02:00 [INF] LASTEXCEPTION:eyJXaGVuIjoiMjAyNC0wNy0yMlQyMjo1NzoxNy4yOTQ0NDgrMDI6MDAiLCJJbmZvIjoiU3lzdGVtLkFnZ3JlZ2F0ZUV4Y2VwdGlvbjogQSBUYXNrJ3MgZXhjZXB0aW9uKHMpIHdlcmUgbm90IG9ic2VydmVkIGVpdGhlciBieSBXYWl0aW5nIG9uIHRoZSBUYXNrIG9yIGFjY2Vzc2luZyBpdHMgRXhjZXB0aW9uIHByb3BlcnR5LiBBcyBhIHJlc3VsdCwgdGhlIHVub2JzZXJ2ZWQgZXhjZXB0aW9uIHdhcyByZXRocm93biBieSB0aGUgZmluYWxpemVyIHRocmVhZC4gKE9uZSBvciBtb3JlIGVycm9ycyBvY2N1cnJlZC4gKE9uZSBvciBtb3JlIGVycm9ycyBvY2N1cnJlZC4gKEZhaWxlZCB0byBmaW5kIHRoZSBhc3NldCBMb2Rlc3RvbmVHYW1lU3ltYm9sLikpKVxyXG4gLS0tPiBTeXN0ZW0uQWdncmVnYXRlRXhjZXB0aW9uOiBPbmUgb3IgbW9yZSBlcnJvcnMgb2NjdXJyZWQuIChPbmUgb3IgbW9yZSBlcnJvcnMgb2NjdXJyZWQuIChGYWlsZWQgdG8gZmluZCB0aGUgYXNzZXQgTG9kZXN0b25lR2FtZVN5bWJvbC4pKVxyXG4gLS0tPiBTeXN0ZW0uQWdncmVnYXRlRXhjZXB0aW9uOiBPbmUgb3IgbW9yZSBlcnJvcnMgb2NjdXJyZWQuIChGYWlsZWQgdG8gZmluZCB0aGUgYXNzZXQgTG9kZXN0b25lR2FtZVN5bWJvbC4pXHJcbiAtLS0+IFN5c3RlbS5JTy5GaWxlTm90Rm91bmRFeGNlcHRpb246IEZhaWxlZCB0byBmaW5kIHRoZSBhc3NldCBMb2Rlc3RvbmVHYW1lU3ltYm9sLlxyXG5GaWxlIG5hbWU6ICdMb2Rlc3RvbmVHYW1lU3ltYm9sJ1xyXG4gICBhdCBEYWxhbXVkLlN0b3JhZ2UuQXNzZXRzLkRhbGFtdWRBc3NldE1hbmFnZXIuPD5jX19EaXNwbGF5Q2xhc3MxOV8wLjw8Q3JlYXRlU3RyZWFtQXN5bmM+Z19fQ3JlYXRlSW5uZXJBc3luY3wxPmQuTW92ZU5leHQoKSBpbiBDOlxcZ29hdHNvZnRcXGNvbXBhbnlzZWNyZXRzXFxkYWxhbXVkXFxcXFN0b3JhZ2VcXEFzc2V0c1xcRGFsYW11ZEFzc2V0TWFuYWdlci5jczpsaW5lIDI0OVxyXG4gICAtLS0gRW5kIG9mIGlubmVyIGV4Y2VwdGlvbiBzdGFjayB0cmFjZSAtLS1cclxuICAgYXQgRGFsYW11ZC5TdG9yYWdlLkFzc2V0cy5EYWxhbXVkQXNzZXRNYW5hZ2VyLjw+Y19fRGlzcGxheUNsYXNzMTlfMC48PENyZWF0ZVN0cmVhbUFzeW5jPmdfX0NyZWF0ZUlubmVyQXN5bmN8MT5kLk1vdmVOZXh0KCkgaW4gQzpcXGdvYXRzb2Z0XFxjb21wYW55c2VjcmV0c1xcZGFsYW11ZFxcXFxTdG9yYWdlXFxBc3NldHNcXERhbGFtdWRBc3NldE1hbmFnZXIuY3M6bGluZSAyNzdcclxuICAgLS0tIEVuZCBvZiBpbm5lciBleGNlcHRpb24gc3RhY2sgdHJhY2UgLS0tXHJcbiAgIC0tLSBFbmQgb2YgaW5uZXIgZXhjZXB0aW9uIHN0YWNrIHRyYWNlIC0tLSIsIkNvbnRleHQiOiJVbm9ic2VydmVkIGV4Y2VwdGlvbiBpbiBUYXNrLiJ9
2024-07-22 22:57:17.245 +02:00 [ERR] Unobserved exception in Task.
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (One or more errors occurred. (One or more errors occurred. (Failed to find the asset LodestoneGameSymbol.)))
 ---> System.AggregateException: One or more errors occurred. (One or more errors occurred. (Failed to find the asset LodestoneGameSymbol.))
 ---> System.AggregateException: One or more errors occurred. (Failed to find the asset LodestoneGameSymbol.)
 ---> System.IO.FileNotFoundException: Failed to find the asset LodestoneGameSymbol.
File name: 'LodestoneGameSymbol'
   at Dalamud.Storage.Assets.DalamudAssetManager.<>c__DisplayClass19_0.<<CreateStreamAsync>g__CreateInnerAsync|1>d.MoveNext() in C:\goatsoft\companysecrets\dalamud\\Storage\Assets\DalamudAssetManager.cs:line 249
   --- End of inner exception stack trace ---
   at Dalamud.Storage.Assets.DalamudAssetManager.<>c__DisplayClass19_0.<<CreateStreamAsync>g__CreateInnerAsync|1>d.MoveNext() in C:\goatsoft\companysecrets\dalamud\\Storage\Assets\DalamudAssetManager.cs:line 277
   --- End of inner exception stack trace ---
   --- End of inner exception stack trace ---
2024-07-22 22:57:17.614 +02:00 [INF] Lumina is ready: C:\Program Files (x86)\Steam\steamapps\common\FINAL FANTASY XIV Online\game\sqpack
2024-07-22 22:57:17.723 +02:00 [INF] [PLUGINR] Successfully fetched repo: https://kamori.goats.dev/Plugin/PluginMaster
2024-07-22 22:57:17.725 +02:00 [INF] [PLUGINR] Fetching repo: https://raw.githubusercontent.com/lichie567/LMeter/main/repo.json
2024-07-22 22:57:17.786 +02:00 [INF] [PLUGINR] Successfully fetched repo: https://raw.githubusercontent.com/lichie567/LMeter/main/repo.json
2024-07-22 22:57:17.786 +02:00 [INF] [PLUGINM] PluginMasters reloaded, now refiltering...
2024-07-22 22:57:19.938 +02:00 [ERR] Failed to get ReShade 6 DXGISwapChain::on_present offset!
2024-07-22 22:57:19.939 +02:00 [INF] Hooking present for ReShade with fallback 5.X sig
2024-07-22 22:57:19.968 +02:00 [INF] ReShade DLL: C:\Program Files (x86)\Steam\steamapps\common\FINAL FANTASY XIV Online\game\dxgi.dll (crosire's ReShade post-processing injector for 64-bit - 6.2.0.1878) with DXGISwapChain::on_present at 0x7FFAD2C3DFDA(C:\Program Files (x86)\Steam\steamapps\common\FINAL FANTASY XIV Online\game\dxgi.dll+0x13DFDA(.text+0x13CFDA))
2024-07-22 22:57:23.224 +02:00 [INF] [PLUGINM] ============= LoadPluginsSync(FrameworkTickSync) START =============
2024-07-22 22:57:23.224 +02:00 [INF] [PLUGINM] ============= LoadPluginsSync(FrameworkTickSync) END =============
2024-07-22 22:57:23.224 +02:00 [INF] [PLUGINM] ============= LoadPluginsAsync(FrameworkTickAsync) START =============
2024-07-22 22:57:23.224 +02:00 [INF] [PLUGINM] ============= LoadPluginsAsync(FrameworkTickAsync) END =============
2024-07-22 22:57:23.339 +02:00 [WRN] [HITCH] Long FrameworkUpdate detected, 166.7633ms > 50ms - check in the plugin stats window.
```

**Comment by nopjmp** [comment]
6.2.0 is the only affected version (released 2024-07-21), lower versions are fine for 6.x

**Comment by anya-hichu** [comment]
Experiencing the same problem with 6.2 and it seems related to this recent change https://github.com/goatcorp/Dalamud/commit/3215b6dddf5da60fa9a7b634f8511a424d01f939

**Comment by shadowkras** [comment]
This issue is also happening with gshade. Reinstalling either applications doesn't fix it.

**Comment by reiichi001** [comment]
The current release of gshade is based on reshade 6.2.0. 

Reinstalling it won't fix this because the new version broke. Your options are uninstall or downgrade (if available, but gshade removes old installers preventing this)

**Comment by KeeperoftheKeys** [comment]
I'm also having this problem Dalamud isn't loading at all and I'm using ReShade. I just updated it to version 6.2.0 as I needed to add more of the shaders so I just went ahead with it.

**Comment by reiichi001** [comment]
> I'm also having this problem Dalamud isn't loading at all and I'm using ReShade. I just updated it to version 6.2.0 as I needed to add more of the shaders so I just went ahead with it.

yes, as previously stated, reshade 6.2.0 (and any forks based on it) will break imgui (dalamud/plugin UI). Your options are to:
1. Switch to Dalamud Testing (or wait for the stable release - we're still working on it)
2. Downgrade reshade back to 6.1.1 or lower
3. Disable reshade until the dalamud update goes out

**Comment by KeeperoftheKeys** [comment]
> > I'm also having this problem Dalamud isn't loading at all and I'm using ReShade. I just updated it to version 6.2.0 as I needed to add more of the shaders so I just went ahead with it.
> 
> yes, as previously stated, reshade 6.2.0 (and any forks based on it) will break imgui (dalamud/plugin UI). Your options are to:
> 
> 1. Switch to Dalamud Testing (or wait for the stable release - we're still working on it)
> 2. Downgrade reshade back to 6.1.1 or lower
> 3. Disable reshade until the dalamud update goes out

Yeah I decided to disable it instead as I see don't really know how to install previous versions.

**Comment by tomcaret** [comment]
For me Dalamud works with Reshade 6.2.0 but not with XIVAlexander.
All three were working as of yesterday but today (after I changed nothing, and nothing was updated) they stopped working together. 
First I removed Reshade but Dalamud was still not working. Removing XIVAlexander made Dalamud working again and I enabled Reshade (6.2.0) again and both of them work together.
